app/helpers/views_helper.rb

```diff
@@ -0,0 +1,9 @@
+module ViewsHelper
+  def get_headings(kase, correspondence_type)
+    if kase.current_state == "rejected" && correspondence_type.abbreviation == "OFFENDER_SAR"
+      content_tag(:span, t("cases.new.offender_sar.rejected"), class: "page-heading--secondary")
```

Comment:
Is this the same as doing `t4c(kase, "", "rejected", case_type: kase.pretty_type)`?